.devcontainer/requirements.txt

```diff
@@ -0,0 +1,38 @@
+asdf>=2.10.0
```

Comment:
Is it really needed to duplicate the requirements here?

Does codespaces not support running `pip install -e[all]`?

Reply:
I agree, this can be a postCreateCommand to avoid duplicate requirements that could get out of date.